Week3/homework/step2-3.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
 
+// Test PASS
 // Use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  if (num < 0) {
+    result = '';
```

Comment:
👍 for checking the `num` value in if condition. I would write the same comment from above about the `for` loop though. We need not separate the declaration of `i` outside the for loop. 🙂